src/plugins/workspace/server/plugin.ts

```diff
@@ -157,7 +158,13 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
         enabled: true,
         permissionEnabled: isPermissionControlEnabled,
       },
+      opensearchDashboards: { isDashboardAdmin: true },
     }));
+    // Dynamically update capabilities based on the auth information from request.
+    core.capabilities.registerSwitcher((request) => {
+      const isDashboardAdmin = getWorkspaceState(request).isDashboardAdmin !== false;
```

Comment:
is this redundant ? 

like
```
getWorkspaceState(request).isDashboardAdmin
```

should equal true right? Or could it potentionally not have this boolean? if so then `!!` to cast it to a boolean then get the value

Reply:
Thanks, updated.